### ISSUE TITLE
Cannot read property 'platform' of undefined

### ISSUE THREAD

**Issue by lethevimlet** [theintern/intern#897]
I'm currently using a Selenium Grid on the same LAN as the machine where I try to launch intern 4.1.5 tests,
I can launch with success remote functional tests for Chrome and Edge but for some reason Firefox and Internet Explorer fails with the same error:

![image](https://user-images.githubusercontent.com/2963923/38066580-ea906b58-3308-11e8-93ef-3358b881c3c3.png)

Surprisingly even with test failing, Firefox and Internet Explorer actually get executed and a blank window is displayed on the remote machine, but no automation happens.

I'am pretty sure I have the correct webdrivers for each browser and arch and since I had it working on Intern 3, and currently on Intern 4, Chrome and Edge works but only Firefox and Internet Explorer fails, I believe it might be a bug with the current intern release. 

This is my intern.json file:
```
{
    "proxyPort": 9000,
    "proxyUrl": "http://192.168.0.10:9000/",
    "tunnel": "null",
    "tunnelOptions": {
        "host": "192.168.0.5:4444",
        "verbose": true
    },
    "suites": [],
    "functionalSuites": [
        "tests/functional/**/*.js"
    ],
    "environments": [
        {
            "browserName": "firefox"
        }
    ]
}
```
I'm launching selenium server with:
```
java 
-Dwebdriver.chrome.driver=C:/Selenium/drivers/chromedriver.exe 
-Dwebdriver.gecko.driver=C:/Selenium/drivers/geckodriver.exe 
-Dwebdriver.edge.driver=C:/Selenium/drivers/MicrosoftWebDriver.exe 
-Dwebdriver.ie.driver=C:/Selenium/drivers/IEDriverServer.exe
-jar selenium-server-standalone-3.11.0.jar 
-role node -hub http://192.168.0.5:4444/grid/register 
-browser "browserName=MicrosoftEdge,platform=WINDOWS,maxInstances=10" 
-browser "browserName=internet explorer,platform=WINDOWS,maxInstances=10" 
-browser "browserName=chrome,platform=WINDOWS,maxInstances=10" 
-browser "browserName=firefox,platform=WINDOWS,maxInstances=10"
```
And finally this is the test I'm running, a simple open google.com

const { registerSuite } = intern.getInterface("object");
```
registerSuite("Example Suite", {
    testPage: function () {
        return this.remote.get("https://www.google.com");
    }
});
```
I'm using latest Firefox (59.0.2) and geckodriver 0.20.0 with Selenium 3.11.0, and as I all ready stated Intern 4.1.5

Any ideas of what could be causing this behaviour? It seems it's expecting a capability that is not provided but that's just a wild guess.

BTW kudos for the awesome ascii art angry table flip (>^_^)>

**Comment by jason0x43** [comment]
Apparently I need to switch back to GMail, because Inbox makes it way too easy to build up a backlog of messages.

Could you verify what version of Leadfoot is in your install? This issue sounds like https://github.com/theintern/leadfoot/issues/125, and should have been addressed in Leadfoot 2.0.3 in December.

**Comment by lethevimlet** [comment]
My project I'ts currently using intern 4.1.5 with its default Leadfoot version, which I believe to be 2.0.3.

![image](https://user-images.githubusercontent.com/2963923/38782011-af9d4866-40ed-11e8-816a-bbab792b471d.png)

**Comment by jason0x43** [comment]
In reviewing your original post, I see that the `capabilities` object wasn't actually defined during Leadfoot initialization. _That_ issue was fixed a bit more recently (https://github.com/theintern/leadfoot/issues/139), and was published in Leadfoot 2.1.0, which is used by default in Intern 4.2.0. Give that a try.
